go.mod

```diff
@@ -2,6 +2,8 @@ module tailscale.com
 
 go 1.19
 
+replace github.com/u-root/uio v0.0.0-20220204230159-dac05f7d2cb4 => github.com/loongarch64/uio v0.0.0-20220922081955-9136972a7648
```

Comment:
We don't like checking in replace directives. They're too invasive for people who depend on this repo.

Can you upstream your changes to `github.com/u-root/uio` first?

Reply:
https://github.com/u-root/uio/pull/6 
Wait for this to be merged